### DISCUSSION TITLE
dpgen simplify ouputs

### DISCUSSION THREAD

**Discussion by AroundPeking** [deepmodeling/dpgen#1310] [Q&A]
I finished `dpgen simplify`, aiming to select fewer structures from 10009 structures dataset.

However, the output issue I cannot comprehend is why only 5009 structures are simplified here:
[dpgen.log](https://github.com/deepmodeling/dpgen/files/12396343/dpgen.log)

The para.json file is here:
[para.json.log](https://github.com/deepmodeling/dpgen/files/12396373/para.json.log)

Another question is: what is the difference between pick_data and init_data_* in para.json for `simplify`?

**Comment by AroundPeking** [comment]
I found that the `init_pick_number`  were thrown away and the rest were simplified. Right?

**Reply by njzjz** [reply]
They are directly added into the training set.
